### PR TITLE
Upgrade test dependencies

### PR DESCRIPTION
**Pull request by juwara0**
### This project uses [semver](semver.org), please check the scope of this pr:

- [ ] #none# - documentation fixes and/or test additions
- [x] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

# CHANGELOG
* **Updated** testing dependencies
* **Removed** no longer needed `bower.json` and `.bowerrc`

### CODE REVIEW

File: tests/dummy/app/pods/demo/template.hbs [hunk 1]
```diff
@@ -80,13 +80,13 @@
     {{/frost-button}}
   {{/frost-popover}}
 {{/frost-button}}
-<br />
+<br >
```
Comment (notmessenger) [inline tests/dummy/app/pods/demo/template.hbs]: Likely don't need the space before the closing bracket

File: tests/dummy/app/pods/demo/template.hbs [hunk 2]
```diff
@@ -80,13 +80,13 @@
...
 {{#frost-button hook='resizeEnabledButton' size='small' priority='primary' class='resize-on' text='Resize Enabled'}}
   {{#frost-popover target='.resize-on' closest=true resize=true position='left'}}
     <span class='inside'>Extra long content that would require resizing to be visible</span>
   {{/frost-popover}}
 {{/frost-button}}
-<br />
+<br >
```
Comment (notmessenger) [inline tests/dummy/app/pods/demo/template.hbs]: Likely don't need the space before the closing bracket

**Comment by notmessenger** [comment]
👍

File: addon/components/frost-popover.js [hunk 3]
```diff
@@ -221,7 +221,7 @@ export default Component.extend(PropTypeMixin, {
         return viewportProp.call(this, this.getTarget())
     }
   },
-/* eslint-disable complexity */
+  /* eslint-disable complexity */
```
[the remark this answers is not in the document]
Comment (juwara0) [reply addon/components/frost-popover.js]: There were no enables. I went through and fixed this.

File: addon/components/frost-popover.js [hunk 4]
```diff
@@ -390,7 +390,7 @@ export default Component.extend(PropTypeMixin, {
       left
     }
   },
-/* eslint-disable complexity */
+  /* eslint-disable complexity */
```
[the remark this answers is not in the document]
Comment (juwara0) [reply addon/components/frost-popover.js]: There were no enables. I went through and fixed this.

File: tests/dummy/app/pods/demo/template.hbs (hunk 1 above)
Comment (juwara0) [reply tests/dummy/app/pods/demo/template.hbs]: Fixed

**Comment by juwara0** [comment]
👍

**Pull request merged**